SPECS/xcp-ng-xapi-storage.spec

```diff
@@ -62,6 +62,9 @@ make
 cd build
 %make_install
 
+%post
+systemctl daemon-reload
```

Comment:
Usually you'd use the `%systemd_post` macro (see commented example below).

Reply:
But examples are for a given service not a systemd command. So I should do `%systemd_post daemon-reload` right?

Reply:
And in fact it is more:
```
%post
%systemd_post daemon-reload
```

Reply:
You should check what `%systemd_post` evaluates to.

Reply:
See also https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/#_scriptlets

Reply:
Oh didn't know about rpm --eval.Thx

Reply:
Why do we need this call?  My understanding was that it was for `qemuback.service`, so should indeed be `%post datapath-qemudisk` and commented out

Reply:
we don't as qemuback.service has been removed (commented). See my comment about the fact that I missed that qemuback.service is not shipped anymore.